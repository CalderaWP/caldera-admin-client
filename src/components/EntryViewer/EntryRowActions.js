import React from "react";
import PropTypes from 'prop-types';
import {Button} from '@wordpress/components'

export const EntryRowActions = (props) => {

		return(
			<div>
				<Button
					onClick={props.onView}
				>
					View
				</Button>
				<Button
					onClick={props.onDelete}

				>
					Delete
				</Button>
				<Button
					onClick={props.onResend}

				>
					Resend
				</Button>
			</div>
		)





};

EntryRowActions.propTypes = {
	onView: PropTypes.func,
	onDelete: PropTypes.func,
	onResend: PropTypes.func
};

