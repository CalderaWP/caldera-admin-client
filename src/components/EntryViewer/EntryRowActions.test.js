import renderer from "react-test-renderer";
import React from 'react';
import {EntryRowActions} from "./EntryRowActions";


describe( 'Entry Row actions component', () => {
	it( 'matches snapshot', () => {
		expect(
			renderer.create(
				<EntryRowActions
					onView={() => {}}
					onResend={() => {}}
					onDelete={() => {}}
				/>
			).toJSON()
		).toMatchSnapshot()
	})
});