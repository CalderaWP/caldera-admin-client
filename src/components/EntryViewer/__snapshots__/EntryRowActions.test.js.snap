// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entry Row actions component matches snapshot 1`] = `
<div>
  <button
    autoFocus={undefined}
    className="components-button"
    disabled={undefined}
    onClick={[Function]}
    type="button"
  >
    View
  </button>
  <button
    autoFocus={undefined}
    className="components-button"
    disabled={undefined}
    onClick={[Function]}
    type="button"
  >
    Delete
  </button>
  <button
    autoFocus={undefined}
    className="components-button"
    disabled={undefined}
    onClick={[Function]}
    type="button"
  >
    Resend
  </button>
</div>
`;
